README.md

```diff
@@ -1 +1,25 @@
-# authLab
+## authLab
+
+### Author: Cas Ibrahim
+
+***
+
+#### Overview
+* authLab is a server designed to encrypt user-input passwords and provide authentication to established users
+* authlab utilizes a mongo database to securely store users and encrypted passwords
+
+#### Setup
+* clone the application
+* run npm i to install the following dependencies:
+    * base-64
+    * bcrypt
+    * dotenv
+    * express
+    * jest
+    * jsonwebtoken
+    * mongoose
+
```

Comment:
Something else that I would love to see added is what you are going to need in the .env file!